src/Approximations/template_directions.jl

```diff
@@ -330,3 +330,86 @@ end
 
 end # @eval
 end # if
+
+# ==================================================
+# Polar directions
+# ==================================================
+
+"""
+    PolarDirections{N<:AbstractFloat} <: AbstractDirections{N}
+
+Polar directions representation.
+
+### Fields
+
+- `Nφ`    -- length of the partition of the polar angle
+- `stack` -- list of computed directions
+
+### Notes
+
+The `PolarDirections` constructor provides a sample of the unit sphere
+in ``\\mathbb{R}^2``, which is parameterized by the polar angles
+``φ ∈ Dφ := [0, 2π]`` respectively, see the wikipedia
+entry [Polar coordinate system](https://en.wikipedia.org/wiki/Polar_coordinate_system).
+The domain ``Dφ`` is discretized in ``Nφ`` points.
+Then the Cartesian componentes of each direction are obtained with
+
+```math
+[cos(φᵢ), sin(φᵢ)].
+```
+
+### Examples
+
+The integer passed as an argument is used to discretize ``φ``:
+
+```jldoctest spherical_directions
+julia> using LazySets.Approximations: PolarDirections
+
+julia> pd = PolarDirections(3)
+PolarDirections{Float64}(3, Array{Float64,1}[[1.0, 0.0], [-1.0, 1.22465e-16]])
```

Comment:
I count only two elements here :confused:

Reply:
With `Nφ = 3` you get `φ = 0` and `φ = pi`, since `φ = 2pi` is discarded.

Reply:
I think you should not consider this duplicate in the count.

Reply:
Which count?

Reply:
`Nθ`

Reply:
Oh, you'd like that `PolarDirections(k)` returns exactly `k` vectors -- yes we can change that behavior if we let `φ = Compat.range(N(0.0), N(2*pi), length=Nφ+1)`. I'm fine either way.

For `SphericalCoordinates` it's less clear -- the length is `(Nφ-1)(Nθ-2) + 2`.

Reply:
I did the change.